roles/cifmw_cephadm/defaults/main.yml

```diff
@@ -66,7 +66,7 @@ cifmw_cephadm_pacific_filter: "16.*"
 # The path of the rendered rgw spec file
 cifmw_ceph_rgw_spec_path: /tmp/ceph_rgw.yml
 cifmw_ceph_mds_spec_path: /tmp/ceph_mds.yml
-cifmw_ceph_rgw_keystone_ep: "keystone-internal.openstack.svc:5000"
+cifmw_ceph_rgw_keystone_ep: "https://keystone-internal.openstack.svc:5000"
```

Comment:
`https` here is added to the internal keystone endpoint: if tls is not enabled at Pod level (`podLevel: true` in the ctlplane) but the ingress is configured as an edge router [1] this will fail.
While I think https should be the default, we should isolate the prefix in it's own variable.
I'm ok to follow up on that, but I would try something like:

```
cifmw_ceph_rgw_keystone_ep: "{{ cifmw_ceph_rgw_keystone_urischeme | default("https") }}:keystone-internal.openstack.svc:5000"
```
so we can make it configurable w/o introducing extra complexity.

[1] https://www.redhat.com/architect/encryption-secure-routes-openshift

Reply:
However, considering you can override the entire variable, my comment is not a blocker for the patch, more of a consideration.
So let me know if you want to keep it as it is and I can approve.